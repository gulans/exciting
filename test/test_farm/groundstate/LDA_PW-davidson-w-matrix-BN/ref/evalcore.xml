<?xml version="1.0" encoding="UTF-8"?>
<evalcore>
  <species is="1" chemicalSymbol="N">
    <atom ia="1">
      <state ist="1" n="1" l="0" k="0" eigenvalue="-1.374137325448e1"/>
    </atom>
  </species>
  <species is="2" chemicalSymbol="B">
    <atom ia="1"/>
  </species>
</evalcore>
