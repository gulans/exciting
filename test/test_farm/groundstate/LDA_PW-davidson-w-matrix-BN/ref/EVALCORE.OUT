
Species :    1 (N), atom :    1
 n =  1, l =  0, k =  1 :   -13.74137325    

Species :    2 (B), atom :    1
